### PR TITLE
Add ENS Registry Address to each chain

### LINKED ISSUE #119: Add ENS Registry to each chain
In case the ENS registry is present on the Chain, then its address should be returned:

```javascript
GET /api/v1/chains/

{
  ...
  "ensRegistryAddress": "<checksummed-address>" // optional
}
```

@gnosis/safe-android @gnosis/safe-ios @gnosis/safe-web

### PR DESCRIPTION
**Pull request by fmrsabino**
Closes #119 

- Add `ens_registry_address` to `Chain`
- The `ens_registry_address` can either be `None` or needs to be a checksummed Ethereum address (other values are not valid)
- Adds Web3 dependency (in order to validate the Ethereum addresses)

### CODE REVIEW

File: src/chains/models.py [hunk 1]
```diff
@@ -36,6 +37,9 @@ class Chain(models.Model):
     )
     gas_price_oracle_url = models.URLField(blank=True, null=True)
     gas_price_oracle_parameter = models.CharField(blank=True, null=True, max_length=255)
+    ens_registry_address = models.CharField(
```
Comment (Uxio0) [inline src/chains/models.py:40]: Use `EthereumAddressField` from `gnosis.eth.django.model` (gnosis-py)

**Review by Uxio0 — CHANGES_REQUESTED** [review]

File: src/chains/models.py (hunk 1 above)
Comment (fmrsabino) [reply src/chains/models.py]: Using the `EthereumAddressField` now e3604bc

File: requirements-dev.txt [hunk 2]
```diff
@@ -7,3 +7,4 @@ flake8==3.9.2
 isort==5.9.1
 pre-commit==2.13.0
 pytest-django==4.4.0
+web3==5.20.1
```
Comment (fmrsabino) [inline requirements-dev.txt:10]: Moved the `web3` dependency to the `requirements-dev.txt` as it is only being used directly with the unit tests (to generate addresses)

Comment (Uxio0) [reply requirements-dev.txt:10]: I like this, but gnosis-py requires web3 and will install it, so no difference 😉

**Review by Uxio0 — APPROVED** [review]

**Pull request merged**